XpressAI/xircuits
# 🔢 Support Multiple Types Port

# Description

This pull request adds a few features and fixes on  #201 to support multiple types port.

## References

PR #201 

## Pull Request Type


## Type of Change


# Tests

  1. Create a component and set the port input type to multiple types  ex `a: InArg[Union[float, int]]` 
  2. Link literals of the set types to the port, it should work.
  3. Link literals other than the set types to the port, it should not work. 
  4. Check the error message showing the accepted multitype.
  Note:  You can get the Union functionality by adding `from typing import Union` to your components script


## Tested on?
